src/components/quiz-question-display/quiz-question-display.tsx

```diff
@@ -1,22 +1,36 @@
 import { useGlobal } from 'reactn'
-import { StyleSheet, View, Text } from 'react-native'
+import Animated from 'react-native-reanimated'
+import { StyleSheet, View, Text, Dimensions } from 'react-native'
 import React from 'react'
 
-import { MenuContainer } from '../menu-container'
-import { AnimatedMenuContainer } from '../animated-menu-container'
+import { colors } from '../../styles'
 import { sizes } from '../../styles'
 
 import { useFlashDisplay } from './utils'
 
 const styles = StyleSheet.create({
-  view: {
+  animated: {
```

Comment:
There's some duplication here now which  could probably be addressed if there was another push to unify the menu code. The parts of this which are common with the menu's could be composable between them all somehow.

Reply:
NIT recorded in open issue to be addressed later: https://github.com/js-jslog/harpnative/issues/56#issuecomment-673620700